.gitignore

```diff
@@ -29,3 +29,4 @@ cpptests
 /pennylane_lightning/*.pyd
 /pennylane_lightning/src/Kokkos/
 /pennylane_lightning/src/GBenchmarks/
+prototypes/
```

Comment:
Is this supposed to be pushed to the repository? Is there a convention about this/the name `prototypes` agreed upon for local prototypes? :)

Reply:
Yes. It is. We were lacking a place to implement prototypes, tests and benchmarks. This is for that.

Reply:
Brilliant, thanks! :)